microservices/recordings/src/Model/RawRecordingInfo.php

```diff
@@ -4,49 +4,46 @@
 
 class RawRecordingInfo
 {
-    protected string $filename;
-    protected ?string $callid = null;
-    protected ?string $hashid = null;
+    protected string $fileName;
+    protected string $hashid;
 
     public function __construct(
-        protected string $file,
+        protected string $fullFileName,
+        protected string $callid,
+        protected int    $size,
+        protected int    $age,
     ) {
-
-        $this->filename = basename($this->file);
-
-        if (preg_match("/\w+-\w+-(.*)-\w+-mix.wav/", $this->filename, $matches)) {
-            $this->callid = urldecode($matches[1]);
-            $this->hashid = substr(md5($this->callid), 0, 8);
-        }
+        $this->fileName = basename($this->fullFileName);
```

Comment:
Based on the line below of how to calculate $hashid: 

      $this->fileName = basename($fullFileName);